llama_stack/distribution/datatypes.py

```diff
@@ -234,6 +234,17 @@ class AuthenticationConfig(BaseModel):
     )
 
 
+class QuotaSqliteConfig(BaseModel):
+    db_path: str = Field(default="./quotas.db", description="Path to the SQLite DB file")
+    quota_requests_per_day: int = Field(default=1000, description="Maximum requests per client per window")
+    quota_window_seconds: int = Field(default=86400, description="Quota window length in seconds")
```

Comment:
Should we go with hours? Seconds seems very granular, no?

Reply:
We can introduce a `QuotaLimit` class to flexibly control the `period` and `max_requests`. However, for the `class QuotaMiddleware` itself, I think it’s better to expose the window as `window_seconds` to allow precise, granular control when setting the expiration time in `kv.set()`.